guide/C/ch_invest.xml

```diff
@@ -1605,7 +1605,7 @@ Income
             <row>
               <entry>Assets:Brokerage Account:Stock:AMZN</entry>
 
-              <entry>-100</entry>
+              <entry>&mdash;100</entry>
```

Comment:
I think the best way would be to reset, fix everything up, and re-commit. If you reorganize like I suggest in the general comment then the changes will be a bit messy and there's no real need to preserve that for posterity.